packages/fluentui/docs/src/examples/components/Card/Usage/CardExampleFocusableChildren.tsx

```diff
@@ -0,0 +1,33 @@
+import {
+  Button,
+  Card,
+  cardMultipleFocusableBehavior,
+  DownloadIcon,
+  Flex,
+  MoreIcon,
+  StarIcon,
+  Text,
+} from '@fluentui/react-northstar';
+import * as React from 'react';
+
+const CardExampleFocusableChildren = () => {
+  return (
+    <Card accessibility={cardMultipleFocusableBehavior} aria-roledescription="card with action buttons">
```

Comment:
Naming looks a bit confusing:
- example is `CardExampleFocusableChildren`
- behavior is `cardMultipleFocusableBehavior`

Should be behavior named as `cardChildrenFocusableBehavior`?